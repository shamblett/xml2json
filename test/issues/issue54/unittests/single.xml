<?xml version="1.0" encoding="utf-8"?>
<root>This is a test of a single text node.</root>
