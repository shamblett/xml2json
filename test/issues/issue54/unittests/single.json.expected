{"root":"This is a test of a single text node."}
